libs/src/services/ethContracts/governanceClient.js

```diff
@@ -394,6 +395,26 @@ class GovernanceClient extends ContractClient {
       blockNumber: voteEvent.blockNumber
     }
   }
+
+  /**
+   *
+   * @param {Integer} proposalId id of the governance proposal
+   * @returns {BigNumber} amount of tokens required to reach quorum
```

Comment:
This is a BN not a BigNumber (they're two diff libraries lol)
https://github.com/indutny/bn.js/
https://github.com/MikeMcl/bignumber.js/

Integer should be number also in js land

Reply:
this is always so annoying good find

Reply:
fixed